src/routes/search/containers/SearchPage.js

```diff
@@ -54,6 +83,50 @@ export class SearchPage extends React.Component<PropsType, LocalStateType> {
 
   onFilterTextChange = (filterText: string) => this.setState({filterText: filterText})
 
+  renderFeedback = (categories: Array<CategoryListItemType>) => {
+    const {filterText} = this.state
+    const {t, cities, city, language, route, pathname, feedbackType} = this.props
+
+    if (categories.length !== 0) {
+      return (
+        <React.Fragment>
+          {filterText && (
+            <FeedbackButton>
+              <FeedbackLink to={`${pathname}?feedback=down`}>{t('informationNotFound')}</FeedbackLink>
+            </FeedbackButton>
+          )}
+          <FeedbackModal
+            query={filterText}
+            city={city}
+            cities={cities}
+            language={language}
+            route={route}
+            pathname={pathname}
+            isPositiveRatingSelected={feedbackType === POSITIVE_RATING}
+            isOpen={feedbackType !== undefined}
```

Comment:
Why do you need to check for undefined here?

I think we shouldn't make a difference between null and undefined in our code. So just go with !!feedbackType